fictadvisor-api/src/v2/utils/documentation/telegramGroup/getTelegramGroups.ts

```diff
@@ -0,0 +1,19 @@
+import { ApiDocumentationParams } from 'src/v2/utils/documentation/decorators';
+import { DefaultUnauthorizedResponse } from '../defaultResponses';
+import { TelegramGroupsResponse } from '@fictadvisor/utils';
+
+export const TelegramGroupDocumentationGetTgGroups: ApiDocumentationParams = {
+  isAuth: true,
```

Comment:
name of the documentation object must match such format: `{EntityName}Documentation{MethodName}`